### DISCUSSION TITLE
Is Drag and Drop not functional in Linux desktop?

### DISCUSSION THREAD

**Discussion by pzs7602** [AvaloniaUI/Avalonia#14778] [Q&A] [answered]
I'm using Avalonia in my Desktop application, the drag and drop is working in macOS, but not functional in Ubuntu 22.04, my Avalonia is 11.0.9. I also try the drag and drop sample from: https://github.com/AvaloniaUI/Avalonia.Xaml.Behaviors/tree/master/samples/DragAndDropSample
the drag and drop also not working in ubuntu version, Does anyone meet the same issue?

**Comment by thevortexcloud** [comment, marked as answer]
It has not been implemented on Linux.

https://github.com/AvaloniaUI/Avalonia/issues/5273

There is a PR to add it but it's been sitting there for awhile now. So I am not sure what's happening with it.

https://github.com/AvaloniaUI/Avalonia/pull/13459
